### ISSUE TITLE
Option to choose input mode

### ISSUE THREAD

**Issue by igorasantos** [JoaoVSouto/daily-scrum-maker#4]
In the first 3 input fields (`didLastExpedient`, `difficulties`, `doingToday`), it's common to write the activities based or on the projects we're working or on the type of activities we made or even based on a mix of both.

For instance:
1. Generic mode (as it is now):
- What I did last workday?
`text`
- Difficulties?
`text`
- What do I plan do today?
`text`
- How am I?
`text`
2. Based on projects:
- What I did last workday?
  - projectName1
`text`
  - projectName2
`text`
- and so on
3. Based on type of activities (less common):
- What I did last workday?
  - typeOfActivity (like 'meetings')
`text`
- and so on
4. Based on projects and type of activities:
- What I did last workday?
  - projectName1
`text`
  - typeOfActivity (like 'Meetings')
`text`
- and so on

I wonder if more flexibility into this form idea, would give a better UX.

Maybe adding some kind of option (with toggle button, e.g.) to the side bar (as there is for toggle the theme), leting the user choose if want to write on a generic mode or based on a generic grouping mode.

And when the toggle would be checked by the user, the form schema would be like the item 4 on the list above.

**TL;DR:**
_What if there is a toggle button to subdivide the first 3 input fields?_

What do you think about?

**Comment by marcospb19** [comment]
Looks good to me! :+1:

**Comment by JoaoVSouto** [comment]
What a wonderful suggestion @igorasantos, I really appreciated it! Looking forward to implement this feature as soon as I can. Thank you! :smile:

**Comment by igorasantos** [comment]
I'm glad you liked! :smiley:

**Comment by JoaoVSouto** [comment]
After 1001 years, I finally finished it :smile: 

> Before late than never!
